include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.h

```diff
@@ -654,9 +652,9 @@ class SpatialOperatorBase : public dealii::Subscriptor
   mutable VectorType const * pressure_ptr;
 
   /*
-   * LES turbulence modeling.
+   * LES turbulence and generalized Newtonian models.
```

Comment:
similar comment as above: Imagine we add another type of variable viscosity apart from LES turbulence model or generalized Newtonian. Then, we would have to update multiple comments. So I think it's best to just generalize the comment here towards variable viscosity without any further details.